README.md

```diff
@@ -194,14 +194,10 @@ build the library binary manually. This can be done with the following command `
 
 ### Cloning the repository
 
-Clone recursively to pull-in needed submodule(s):
```

Comment:
If you don't explicitly tell most users *why* they must do this, they'll consider it an "extra unnecessary step" when in fact it's not. This statement is proven by experience.

Reply:
The --recursive would not be needed anymore if it is executed in the cmake process. I removed it and made the statement clearer.